File: PRAK503-2510817320004-RahmaSyaritaJaya.c
#include <stdio.h>
#include <math.h>

int maksimal(int a, int b){
    return fmax(a, b);
}
int minimal(int a, int b){
    return fmin(a, b);
}

int main(){
    int batas = 0, maks = 0, minim = 0;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    }
    printf("%d %d", maks, minim);
    return 0;
}

File: PRAK503-2510817320004-RahmaSyaritaJaya.py
def maksimal(a, b):
    return max(a, b)
def minimal(a, b):
    return min(a, b)

bilangan = int(input())
batas, maks, minim = 1, 0, 0

if bilangan > 0:
    angka_int = list(map(int, input().split()))
    while batas < bilangan:
        maks = maksimal(maks, angka_int[batas])
        minim = minimal(minim, angka_int[batas])
        batas += 1
print(maks, minim)